### PR TITLE
fix(core): provide screen reader access for inline help

### LINKED ISSUE #4219: Inline help is not read by Screen Reader
#### Is this a bug, enhancement, or feature request?
bug
#### Briefly describe your proposal.
Screen readers do not announce the existence of the inline help nor the content it holds.

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
latest v0.26.0-rc.21

#### If this is a bug, please provide steps for reproducing it.
1. Go to https://fundamental-ngx.netlify.app/#/core/inlineHelp 
2. Turn VoiceOver on(in Mac) or any other screen reader app.
3. Navigate through the inline help examples. Screen reader does not announce the existence of the inline help nor the content it holds.

#### Is there anything else we should know?
also linked to issues #3440 #3441

### PR DESCRIPTION
**Pull request by sKudum**
#### Please provide a link to the associated issue.
Closes #4219 

#### Please provide a brief summary of this pull request.
screen reader will read the content in inline help on focus using mouse or tab.

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
- [x] Run npm run build-pack-library and test in external application

Documentation checklist:
- [n/a] update `README.md`
- [n/a] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [n/a] Documentation Examples
- [n/a] Stackblitz works for all examples

### CODE REVIEW

**Review by InnaAtanasova — APPROVED** [review]

File: libs/core/src/lib/inline-help/inline-help.directive.ts [hunk 1]
```diff
@@ -18,7 +18,7 @@ export class InlineHelpDirective extends BasePopoverClass implements OnInit, OnC
     /** The trigger events that will open/close the inline help component.
      *  Accepts any [HTML DOM Events](https://www.w3schools.com/jsref/dom_obj_event.asp). */
     @Input()
-    triggers: string[] = ['mouseenter', 'mouseleave'];
+    triggers: string[] = ['mouseenter', 'mouseleave', 'focus', 'focusout'];
```
Comment (InnaAtanasova) [inline libs/core/src/lib/inline-help/inline-help.directive.ts:21]: shouldn't it be `focusin` instead of `focus` ?

**Review by Betrozov — CHANGES_REQUESTED** [review]
please remove package-lock.json

**Review by aberikashvili — APPROVED** [review]
LGTM

**Review by Betrozov — APPROVED** [review]

**Comment by Seamoo13** [comment]
tested using chrome screen reader extension. for me everything works ok. But I cannot test using JAWS. @puru-hk could you also take a look?

**Review by Seamoo13 — APPROVED** [review]

**Comment by puru-hk** [comment]
Tested in Jaws 2021 it is working as expected.

**Review by puru-hk — APPROVED** [review]

**Pull request merged**